prost-derive/Cargo.toml

```diff
@@ -20,7 +20,7 @@ proc_macro = true
 
 [dependencies]
 anyhow = "1.0.1"
-itertools = { version = ">=0.10, <=0.12", default-features = false, features = ["use_alloc"] }
-proc-macro2 = "1"
+itertools = ">=0.10.1, <=0.12"
+proc-macro2 = ">=1.0.60, <2"
```

Comment:
ah that's right, good catch.

Reply:
to note i don't think worrying about the age of dependencies in the proc macro crate is as important here. because this crate has no library it cannot be depended upon except by its observed behavior, so dependency versions shouldn't be able to conflict with any other binaries.

Reply:
I mention the age due to our MSRV policy, I would like to apply that policy also to the dependencies.